vue_app/src/App.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div id="app">
+    <TheHeader />
+    <TheHero />
+    <TheMainContent />
+  </div>
+</template>
+
+<script>
+import TheHeader from "./components/TheHeader.vue";
+import TheHero from "./components/TheHero.vue";
+import TheMainContent from "./components/TheMainContent.vue";
+
+export default {
+  name: "app",
+  components: {
+    TheHeader,
+    TheHero,
+    TheMainContent
+  }
+};
+</script>
+
+<style lang="scss">
+#app {
+  -webkit-font-smoothing: antialiased;
```

Comment:
would recommend looking at autoprefixer when you get the chance, it's amazing! It's typically pretty straightforward to implement in whatever css pipeline you're using too